### DISCUSSION TITLE
Mermaid Render error while using tabset in markdow editor

### DISCUSSION THREAD

**Discussion by lyan2k** [requarks/wiki#4189] [Error / Bug Report]
Environment :
* Wiki.js 2.5.201 on raspbian
* french
* client : windows 10 
* browser : tested on firefox, chrome & edge
* markdown editor

Issue : I try to get a page that displays graphs in tabs. I use the tabset code to crate tabs in my page and use mermaid to create the graphs.
when ! refresh the page I get the tabs and the graph renders correctly on the first tab, but the following tabs (in firefox) show an error message "syntax error in graph". The other browser don't render the graphs.

Sample code : 
`# Tabeset test {.tabset}`
`## Tab 1`
` ``` mermaid `
`graph LR`
`	A[A] --> B[B]`
` ``` `
`## Tab 2`
` ``` mermaid`
`graph LR`
`       C[C] --> D[D]`
` ``` `
`## Tab3`
` ``` mermaid`
`graph LR`
`	 E[E] --> F[F]`
` ``` `

https://wiki.s0laire.fr/fr/PTC/Cocottes/test2

**Comment by francisbrand** [comment]
I have the same issue. Did you find a solution?
